Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -412,20 +455,19 @@ public void Wzhzhzh(EntityUid uid, MicrowaveComponent component, EntityUid? user
                     return;
                 }
 
-                // destroy microwave
                 if (_tag.HasTag(item, "MicrowaveMachineUnsafe") || _tag.HasTag(item, "Metal"))
                 {
-                    component.Broken = true;
-                    SetAppearance(uid, MicrowaveVisualState.Broken, component);
-                    _audio.PlayPvs(component.ItemBreakSound, uid);
-                    return;
+                    component.IsSparking = true;
                 }
 
-                if (_tag.HasTag(item, "MicrowaveSelfUnsafe") || _tag.HasTag(item, "Plastic"))
+                if (_tag.HasTag(item, "Plastic"))
```

Comment:
Why was `MicrowaveSelfUnsafe` removed?

Reply:
```
                if (_tag.HasTag(item, "MicrowaveMachineUnsafe") || _tag.HasTag(item, "Metal"))
                {
                    component.IsSparking = true;
                }

                if (_tag.HasTag(item, "Plastic"))
                {
                    var junk = Spawn(component.BadRecipeEntityId, Transform(uid).Coordinates);
                    _container.Insert(junk, component.Storage);
                    QueueDel(item);
                    _sharedContainer.EmptyContainer(component.Storage);
                    _audio.PlayPvs(component.FoodDoneSound, uid, AudioParams.Default.WithVolume(-1));
                    return;
                }
```
unsafe items cause it to explode.

Reply:
apologies, just realized these are literally different things.